plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java

```diff
@@ -294,6 +315,21 @@ public void onStart(Process process) {
     }
   }
 
+  private static final String URL_PATTERN = "http://[a-zA-Z0-9][a-zA-Z0-9-.]*[a-zA-Z0-9]:([0-9]+)";
```

Comment:
I think this needs some documentation that it's specific to the URLs generated by the dev_appserver, where everything is generally hosted on this machine with a port and no paths.  And maybe a less general-sounding variable name?